github-to-zulip/pull-reminders.sh

```diff
@@ -51,11 +58,18 @@ post_to_zulip(){
          -u "${ZULIP_TOKEN}" \
          -d 'type=stream' \
          -d 'to=bot-test' \
-         -d 'include_custom_profile_fields=true' \
          -d 'topic=Pull reminder' \
          -d "content=$1"
 }
 
+github_username_to_zulip_name() {
+    curl --silent --show-error --fail \
+        -X GET \
```

Comment:
Let's use long parameter names `--request`, `--user`, etc

Reply:
I was following the convention already being used in the script. But, fair enough. I'll change this.